modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/auto/deploy/BatchEngineImportAutoDeployListener.java

```diff
@@ -76,9 +77,12 @@ public int deploy(AutoDeploymentContext autoDeploymentContext)
 				ZipEntry zipEntry = iterator.nextElement();
 
 				if (Objects.equals(zipEntry.getName(), "batch-import.json")) {
-					batchImportConfiguration = _objectMapper.readValue(
```

Comment:
hi @shuyangzhou I didn't add try-with-resources for input streams fetched from zipFile.getInputStream as zipFile itself is under try-with-resources and it will close all inputStreams fetched from fetched from zipFile.getInputStream. The same applies for change below